zerver/lib/webhooks/common.py

```diff
@@ -20,8 +20,7 @@
 from zerver.lib.validator import check_list, check_string
 from zerver.models import UserProfile
 
-MISSING_EVENT_HEADER_MESSAGE = """
-Hi there!  Your bot {bot_name} just sent an HTTP request to {request_path} that
+MISSING_EVENT_HEADER_MESSAGE = """Hi there!  Your bot {bot_name} just sent an HTTP request to {request_path} that
```

Comment:
I think for most of the cases where you did this line-wrap, it'd have been better to do `"""\` on the first line, to preserve the readability of the error message paragraph as written without creating an extra newline.